source/extensions/quic_listeners/quiche/platform/quic_pcc_sender_impl.h

```diff
@@ -0,0 +1,32 @@
+#pragma once
+
+// NOLINT(namespace-envoy)
+
+// This file is part of the QUICHE platform implementation, and is not to be
+// consumed or referenced directly by other Envoy code. It serves purely as a
+// porting layer for QUICHE.
+
+#include "common/common/assert.h"
+
+#include "quiche/quic/core/quic_types.h"
+
+namespace quic {
+
+class QuicClock;
+class QuicConnectionStats;
+class QuicRandom;
+class QuicUnackedPacketMap;
+class RttStats;
+class SendAlgorithmInterface;
+
+// Interface for creating a PCC SendAlgorithmInterface.
+SendAlgorithmInterface* CreatePccSenderImpl(const QuicClock* clock, const RttStats* rtt_stats,
```

Comment:
I will say I'm surprised we can't just keep this compiled out.  Maybe some day we can switch more over to a link-in-factory model like Envoy has.